projects/addon-doc/components/navigation/navigation.component.ts

```diff
@@ -165,6 +164,16 @@ export class TuiDocNavigation {
         this.openActivePageGroup();
     }
 
+    protected onFocusSearch(event: KeyboardEvent): void {
+        if (
+            event.code === 'Slash' &&
+            !(this.doc.activeElement instanceof HTMLInputElement)
```

Comment:
Let's check that nothing is focused instead, `this.doc.activeElement === this.doc.body`, just in case.

Reply:
Then we will lose the ability to quickly switch to the search if, for example, a button is in focus. For example, when clicking on Typescript here. Do you really want this?
![image](https://github.com/user-attachments/assets/f4a74be5-abc5-43d6-a0f6-44d31f16bea7)

Reply:
Ok, it's just a demo portal anyway, I doubt there would be any problems because of this